code/modules/catalogue/rewards/equipment_vendor.dm

```diff
@@ -36,7 +36,7 @@
 		new /datum/data/exploration_equipment("Jump Boots",					/obj/item/clothing/shoes/bhop,										250),
 		new /datum/data/exploration_equipment("Luxury Shelter Capsule",		/obj/item/device/survivalcapsule/luxury,							310),
 		new /datum/data/exploration_equipment("Defense Equipment - Smoke Bomb",/obj/item/weapon/grenade/smokebomb,								10),
-		new /datum/data/exploration_equipment("Defense Equipment - Razor Drone Deployer",/obj/item/weapon/grenade/spawnergrenade/manhacks/station,	100),
+		new /datum/data/exploration_equipment("Defense Equipment - Razor Drone Deployer",/obj/item/weapon/grenade/spawnergrenade/manhacks/station/locked,	100),
```

Comment:
//Vorestation edit

Reply:
There is big overarching edit comment for entirety for mining list already